scripts/lint

```diff
@@ -27,6 +27,9 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
         yapf -dpr ${DIRS_TO_CHECK[@]}
         # Lint
         flake8 ${DIRS_TO_CHECK[@]}
-
+        # Type checking
+        mypy src
+            # When https://github.com/stac-utils/stactools/issues/155 is complete
+            # remove --ingore-missing-imports
```

Comment:
I don't think we need this comment as we don't have that flag.

Reply:
Oops. Yeah, that's left over from before I added the `ini` file